<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Left</class>
 <widget class="QWidget" name="Left">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>555</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>304</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Left</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(106, 106, 106);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(106, 106, 106);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>45</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border-image: url(:/images/btnbg1.PNG);
border-color: rgb(191, 191, 191);</string>
        </property>
        <property name="text">
         <string>视频源操作                               </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cameraStartBtn">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;微软雅黑&quot;;
background-color: rgb(80, 80, 80);
color: rgb(255, 255, 255);
border-image: url(:/images/btnbg2.PNG);</string>
        </property>
        <property name="text">
         <string>实时视频                                         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="videoChooseBtn">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;微软雅黑&quot;;
border-image: url(:/images/btnbg2.PNG);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>历史视频                                         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="imgChooseBtn">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;微软雅黑&quot;;
border-image: url(:/images/btnbg2.PNG);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>图　　片                                         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>45</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border-image: url(:/images/btnbg1.PNG);
border-color: rgb(191, 191, 191);</string>
        </property>
        <property name="text">
         <string>操作                                         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="videoStartBtn">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border-image: url(:/images/btnbg2.PNG);</string>
        </property>
        <property name="text">
         <string>开始                                               </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="videoPauseBtn">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;微软雅黑&quot;;
border-image: url(:/images/btnbg2.PNG);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>暂停                                               </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="videoClearBtn">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;微软雅黑&quot;;
border-image: url(:/images/btnbg2.PNG);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>清除                                               </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="registBtn">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>45</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border-image: url(:/images/btnbg1.PNG);
border-color: rgb(191, 191, 191);</string>
        </property>
        <property name="text">
         <string>注册                                         </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
